controls/V-81009.rb

```diff
@@ -0,0 +1,42 @@
+# encoding: utf-8
+#
+control "V-81009" do
+  title "The Red Hat Enterprise Linux operating system must mount /dev/shm with the nodev option."
+  desc  "
+  The \"nodev\" mount option causes the system to not interpret character or block special devices. 
+  Executing character or block special devices from untrusted file systems increases the opportunity 
+  for unprivileged users to attain unauthorized administrative access."
+  impact 0.5
+  tag "gtitle": "SRG-OS-000368-GPOS-00154"
+  tag "gid": "V-81009"
+  tag "rid": "SV-95721r1_rule "
+  tag "stig_id": "RHEL-07-021022"
+  tag "cci": ["CCI-001764"]
+  tag "documentable": false
+  tag "nist": ["CM-6 b", "Rev_4"]
+  tag "subsystems": ['file_system']
+  desc "check", "
+  Verify that the \"nodev\" option is configured for /dev/shm.
+
+  Check that the operating system is configured to use the \"nodev\" option for /dev/shm with the following command:
+
+  # cat /etc/fstab | grep /dev/shm | grep nodev
+
+  tmpfs /dev/shm tmpfs defaults,nodev,nosuid,noexec 0 0
+
+  If the \"nodev\" option is not present on the line for \"/dev/shm\", this is a finding.
+
+  Verify \"/dev/shm\" is mounted with the \"nodev\" option:
+
+  # mount | grep \"/dev/shm\" | grep nodev
+
+  If no results are returned, this is a finding.
+  "
+  desc "fix", "
+  Configure the \"/etc/fstab\" to use the \"nodev\" option for all lines containing \"/dev/shm\".
+  "
+
+  describe mount('/dev/shm') do
+    its('options') { should include 'nodev' }
+  end
```

Comment:
This describe statement only takes into consideration what is currently mounted.  It is important to indicate that would be mounted if DR happens (node bounces) and auto-mount procedures take place.  We normally weight everything that is configurable upon 2 scales.  1) configuration (what happens when the node is rebooted) and 2) cached (what parameters/settings are currently affecting the node).   I think in this case you should include the configuration settings.  Here is what that would look like.

```ruby
  describe etc_fstab.where { mount_point == '/dev/shm' } do
    it { should be_configured }
    its('mount_options.flatten') { should include 'nodev' }
  end
```

It is also important to note that `/etc/mtab` also indicates the status of a healthy system.  It is wholly possible for mount to not indicate proper settings (unlikely, but possible).   In that case you might want to take this file into consideration. 

```ruby
  describe etc_fstab('/etc/mtab') { mount_point == '/dev/shm' } do
    it { should be_configured }
    its('mount_options.flatten') { should include 'nodev' }
  end
```

The last possible location to look for 'proper working order' is `/proc/mounts`.   Quote from the mount manual:

> When the proc filesystem is mounted (say at /proc), the files /etc/mtab and /proc/mounts have very similar contents. The former has somewhat more information, such as the mount options used, but is not necessarily up-to-date.

It is also possible that you want to check the cached value in `/proc/mounts` in newer and newer OS versions due to the adoption of systemd `*.mounts` files.